app/javascript/src/Common/components/Spinner.jsx

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import React from 'react'
+
+import './Spinner.scss'
+
+type Props = {
+  size?: 'sm' | 'md' | 'lg' | 'xl',
+  className?: string
+}
+
+const Spinner = ({ size = 'md', className }: Props) => {
+  const classParameters = `pf-c-spinner pf-m-${size.toString()} ${className || ''}`
```

Comment:
Is `toString()` needed if size is already a string?
Why not using default value for `className`:
```suggestion
const Spinner = ({ size = 'md', className = '' }: Props) => {
  const classParameters = `pf-c-spinner pf-m-${size} ${className}`
```